packaging/install.sh

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env bash
 
-echo "###################################################################"
-echo "Cloudify installer is ready!"
-echo "Edit /etc/cloudify/config.yaml, and run cfy_manager install"
-echo "(members of group wheel have write access)"
-echo "###################################################################"
+echo "
+###########################################################################
+Cloudify installer is ready!
+To use it run:
+cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>
+(Use cfy_manager -h for a full list of options)
+
+Alternatively, edit /etc/cloudify/config.yaml to set IPs and other options,
```

Comment:
Replace line with: You can specify the public and private IP addresses as command options or in /etc/cloudify/config.yaml. You can also specify other installation settings in the config.yaml file.
If you specify the public and private IP addresses in the config.yaml file, run:
cfy_manager install

Reply:
Actually, that sounds too verbose, and even a bit more confusing than the original version. Maybe something in the middle?

Reply:
@bmansheim ?

Reply:
You can specify more installation settings in the config.yaml file. If you specify the public and private IP addresses in the config.yaml file, run:
cfy_manager install